python/tvm/relax/testing/relay_translator.py

```diff
@@ -255,6 +261,7 @@ def visit_func(node):
         opt_level=opt_level,
         config=pass_config,
         disabled_pass=disabled_pass,
+        instruments=instruments,
```

Comment:
I like it, though I'm wondering if we should have it forward the instrumentation of the existing `PassContext`, rather than explicitly accept it as an argument.  That way, debug instrumentation set by a much higher scope would also be preserved, without requiring an update to the immediately-calling scope.

```python
instruments = tvm.transform.PassCurrent().instruments
```

Reply:
Thanks for your review @Lunderberg.

I actually did it that way initially, but since the other arguments to `PassContext` such as `opt_level` and `pass_config` were taken as arguments to the relay translator, I modified it take instruments as an argument as well.

I can change it back to be forwarded from current PassContext as you've suggested, either way is fine with me, but in that case, should we also remove `opt_level` and `pass_config` from being arguments and forward them from current context as well? Since that would be a breaking change, I'm a bit hesitant to do that.

Reply:
Hmm, good point.  Since the other arguments are currently specified explicitly, probably better to keep it that way for consistency.

In the long-term, I think that entering a PassContext should probably distinguish between options that override the previous (`opt_level`), options that append to the previous (`instruments`, `disabled_passes`).